ReactNative/ReactTwitter/core/views/deep-linking.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import {
+  StyleSheet,
+  View,
+  Text
+} from 'react-native'
+
+var Button = require('react-native-button')
+var DeepLinkingFacade = require('../service/deep-linking-facade')
```

Comment:
Is there a way to get ride of all these relative path? There is no such thing as a DIR_ROOT variable?